pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.fr-fr.md

```diff
@@ -0,0 +1,150 @@
+- Être connecté à l’[API OVHcloud](https://api.ovh.com/)
+
+## Principes de base
+
+Un snapshot de volume est une copie ponctuelle en lecture seule d'un volume.
+Les snapshots sont créés à partir d'un volume opérationnel existant. Ils ne peuvent exister sans cela.
+
+> [!warning]
+>
+> Veuillez noter qu'une fois qu'un volume est restauré à l'aide d'une copie de snapshot, tous les fichiers ou snapshots créés ultérieurement seront perdus. Lorsqu'un volume est restauré, toutes les données qu'il contient sont remplacées par les données du snapshot. Cette action est irréversible.
+>
+
+Dans ce guide, un volume est également appelé « *share* » comme dans l’API OVHcloud.
+
+## Limites
+
+Il est uniquement possible de restaurer un volume à son snapshot le plus récent disponible. Toutefois, si vous souhaitez restaurer un volume à partir d'un snapshot antérieur, vous devez supprimer les snapshots jusqu'à ce que le snapshot à utiliser pour la restauration soit le plus récent.
+
+## En pratique
+
+### Scénario 1 : restaurer un volume depuis un snapshot de type `manual`
+
+Dans ce scénario, vous souhaitez restaurer votre volume à son dernier snapshot de type `manual`, créé via l'API OVHcloud ou l'espace client OVHcloud.
+
+> [!primary]
+> **Prérequis pour ce scénario :**
+>
+> - Vous avez déjà effectué un snapshot de type `manual`. Dans le cas contraire, vous pouvez créer un snapshot de type `manual` via l’API OVHcloud ou votre espace client OVHcloud.
+> - Le snapshot de type `manual` doit appartenir au volume que vous souhaitez restaurer.
+
+1\. Identifiez le dernier snapshot de volumed e type `manual` à l'aide de l'appel API suivant :
+
+> [!api]
+>
+> @api {GET} /storage/netapp/{serviceName}/share/{shareId}/snapshot
+>
+
+- `{serviceName}` est l'identifiant unique du service
+- `{shareId}` est le volume à restaurer 
+
+![RevertManualSnapshot](images/use_case_1_step_1.png){.thumbnail}
+
+2\. Restaurez votre volume à son dernier snapshot en utilisant l'appel API `/revert` : 
+
+> [!api]
+>
+> @api {POST} /storage/netapp/{serviceName}/share/{shareId}/revert
+>
+
+- `{serviceName}` est l'identifiant unique du service
+- `{shareId}` est le volume à restaurer
+- `{snapshotID}` est le dernier snapshot de volume
+
+L’API OVHcloud retournera uniquement un code HTTP 202 (*Accepted*).<br>
+L'état du volume passera à `reverting`, puis reviendra à `available` une fois le processus de restauration du volume terminé. De manière simultanée, l'état du snapshot passera à `restoring` puis reviendra à `available` une fois le processus de restauration du volume terminé.
+
+![RevertManualSnapshot](images/use_case_1_step_2.png){.thumbnail}
+
+### Scénario 2 : Restaurer un volume depuis un snapshot effectué via la Snapshot policy
+
+Dans ce scénario, une règle de snapshots (*Snapshot policy*) prend des snapshots réguliers (automatiques) d'un volume et vous souhaitez restaurer votre volume au dernier snapshot pris par la stratégie.
```

Comment:
```suggestion
Dans ce scénario, une règle d'une politique de snapshots (*Snapshot policy*) prend des snapshots réguliers (automatiques) d'un volume et vous souhaitez restaurer votre volume au dernier snapshot crée par la *Snapshot policy*.
```

Snapshot policy is translated to *une politique de snapshots* inside [this FR guide](https://help.ovhcloud.com/csm/fr-public-cloud-storage-netapp-snapshot-policies?id=kb_article_view&sysparm_article=KB0059123). A Snapshot Policy can have multiple `rules / régles`, each one taking snapshots at a scheduled interval.